reference: add a mysql system variable windowing_use_high_precision (#2145)
pingcap/docs: reference/configuration/tidb-server/mysql-variables.md

| tx_isolation | GLOBAL \| SESSION | the isolation level of a transaction |
| hostname | NONE | the hostname of the TiDB server |
| max\_execution\_time | GLOBAL \| SESSION |  the execution timeout for a statement, in milliseconds |

> | windowing\_use\_high\_precision | GLOBAL \| SESSION |  whether to compute window operations without loss of precision, default value is ON |

Reviewer: ```suggestion
| windowing\_use\_high\_precision | GLOBAL \| SESSION |  whether to use the high precision mode when computing the window function. The default value is ON |
```
